package.json

```diff
@@ -157,6 +157,7 @@
   },
   "dependencies": {
     "@serverless/cli": "^1.4.0",
+    "ajv": "^6.12.0",
```

Comment:
Let's maintain alphabetical order (it's what npm does when we add/update dependency automatically)

Reply:
Thanks for pointing it out! I didn't know that and I think it happened while I was resolving a merge conflict. Next time I would hopefully keep that in mind

Reply:
Hm.. the security check fails after I manually moved it in https://github.com/serverless/serverless/pull/7335/commits/0b70a124b0ed0e970839fe5a76215ee4a2a37067

Reply:
It appears it's because there's a merge conflict, when you update with master it should be gone

Reply:
It worked! Thanks!